kge/model/embedder/lookup_embedder.py

```diff
@@ -64,7 +64,8 @@ def normalize_embeddings(job):
                             )
                         )
 
-            job.pre_batch_hooks.append(normalize_embeddings)
+            if job.config.get("job.type") == "train":
+                job.pre_batch_hooks.append(normalize_embeddings)
```

Comment:
WOuldn't this change output for evaluation?

Reply:
Perhaps: also do that in the pre_run hook for eval tasks.

Reply:
ok, makes sense